src/frontend/src/lib/components/carousel/Carousel.svelte

```diff
@@ -244,6 +244,12 @@
 		const offset = (currentSlide + 1) * containerWidth;
 		moveSlider({ sliderFrame, animateTo: -offset, withTransition, duration, easing });
 	};
+
+	export const removeSlide = (idx: number) => {
+		slides = slides.filter((_, i) => i !== idx);
```

Comment:
Was just curious - looks like for arrays with length > 100, [filter is faster than slice](https://medium.com/@justintulk/javascript-performance-array-slice-vs-array-filter-4573d726aacb) 🙂

Reply:
yes!!! you never know how many dapps we will have there... hopefully not 100 eheh